### PR TITLE
DUPLO-4007: Attribute replica_collocation_allowed added for duplocloud_duplo_service resource.

### PR DESCRIPTION
**Pull request by tahir-duplo**
## Change description

Attribute replica_collocation_allowed added for duplocloud_duplo_service resource.

## Type of change
- [ ] Bug fix (fixes an issue)
- [ ] New feature (adds functionality)

## Related issues

> Fix [#1]() 

### Development

- [ ] Pull requests may not be submitted to the *master* branch (use *develop* instead) - or they will be closed.
- [ ] Lint rules pass locally
- [ ] Application changes have been tested thoroughly
- [ ] Automated tests covering modified code pass

### Security

- [ ] Security impact of change has been considered
- [ ] Code follows company security practices and guidelines

### Code review 

- [ ] Pull request has a descriptive title and context useful to a reviewer.
- [ ] "Ready for review" label attached and reviewers assigned
- [ ] Changes have been reviewed by at least one other contributor
- [ ] Pull request linked to task tracker where applicable

### CODE REVIEW

**Comment by zafarabbas** [comment]
Task linked: [DUPLO-4007 Add replica_collocation_allowed for duplo service.](https://app.clickup.com/t/8655600/DUPLO-4007)

**Comment by zafarabbas** [comment]
Task linked: [DUPLO-4007 Add replica_collocation_allowed for duplo service.](https://app.clickup.com/t/8655600/DUPLO-4007)

File: duplocloud/resource_duplo_service.go [hunk 1]
```diff
@@ -136,6 +136,12 @@ func duploServiceSchema() map[string]*schema.Schema {
 			Optional:    true,
 			Default:     false,
 		},
+		"replica_collocation_allowed": {
```
Comment (joek-duplo) [inline duplocloud/resource_duplo_service.go:139]: shouldn't we put `Computed: true` here to keep backwards compatibility?

**Review by joek-duplo — CHANGES_REQUESTED** [review]

**Review by joek-duplo — CHANGES_REQUESTED** [review]
`Computed: true`

**Review by joek-duplo — APPROVED** [review]

**Pull request merged**